components/raftstore/src/store/worker/read.rs

```diff
@@ -30,12 +31,12 @@ use crate::store::fsm::store::StoreMeta;
 /// A read only delegate of `Peer`.
 #[derive(Clone, Debug)]
 pub struct ReadDelegate {
-    region: metapb::Region,
+    region: Arc<metapb::Region>,
```

Comment:
Why change it to `Arc<>`?

Reply:
To avoid struct clone for every 'RegionSnapshot'...